pkg/csi_driver/controller.go

```diff
@@ -104,28 +104,26 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 			return nil, status.Error(codes.AlreadyExists, err.Error())
 		}
 	} else {
-
-		// Get the reserved-IPV4-cidr from the request. If value is not provided, reservedIPV4Range is defaulted to ""
+		// If we are creating a new instance, we need to evaluate the reservedIPRange range if reserved-ipv4-cidr is provided
```

Comment:
maybe something like "we need to pick an unused /29 range from reserved-ipv4-cidr" would be more clear